WordPress/Sniffs/WP/AlternativeFunctionsSniff.php

```diff
@@ -126,11 +126,29 @@ public function getGroups() {
 				),
 			),
 
+			'unlink' => array(
+				'type'      => 'warning',
+				'message'   => '%s() is discouraged. Use wp_delete_file() for remote URLs instead.',
```

Comment:
Just wondering: why specifically for "remote URLs" ? And if so, no check is being done in the sniff that the warning will not show for local files. Should that be added ? Or should the `"for remote URLs" be removed ?

Reply:
Removed. Dumb mistake